example-project/src-typescript/api/login/index.ts

```diff
@@ -0,0 +1 @@
+export { isValidLogin } from './login';
```

Comment:
TS example project added for test (all files with  `example-project/src-typescript`root url)

Reply:
Can we clean up this a bit, I mean, it shouldn't be real "working" project to test parser, right, few files is enough. Just trying to prevent codebase growing when it's not necessary (there are ~10 languages, it's gonna grow huge fast). What do you think?

Reply:
Sure, I downloaded a random boilerplate at the beginning just to save time. I did reduce it to ~10 files OK for you @Bogdan-Lyashenko?